portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/ListObjects.tsx

```diff
@@ -1075,7 +1097,7 @@ const ListObjects = ({
                 <input
                   type="file"
                   multiple
-                  onChange={(e) => uploadObject(e)}
+                  onChange={(e) => handleUploadButton(e)}
```

Comment:
```suggestion
                  onChange={handleUploadButton}
```
the arguments are same. so an extra function can be avoided